en/docs/fragments/pickup-dispatch-oidc.md

```diff
@@ -36,7 +36,7 @@ Follow the instructions below to download the sample from GitHub.
 
 1. Navigate to [WSO2 Identity Server Samples](https://github.com/wso2/samples-is/releases).
 
-2. [Download](https://github.com/wso2/samples-is/releases/download/ v4.3.0/pickup-dispatch.war) the `pickup-dispatch.war` file from the latest release assets.
+2. [Download](https://github.com/wso2/samples-is/releases/download/v4.3.0/pickup-dispatch.war) the `pickup-dispatch.war` file from the latest release assets.
```

Comment:
let's include the latest link. refer: https://github.com/wso2/samples-is/tags

Reply:
The latest tags have not been released yet.